packages/snapshot/snapshot.go

```diff
@@ -18,16 +15,15 @@ import (
 	"github.com/iotaledger/goshimmer/packages/ledger/utxo"
 	"github.com/iotaledger/goshimmer/packages/ledger/vm/devnetvm"
 	"github.com/iotaledger/goshimmer/packages/mana"
+	"github.com/iotaledger/goshimmer/packages/notarization"
 )
 
 type Snapshot struct {
 	LedgerSnapshot *ledger.Snapshot
-	ManaSnapshot   *mana.Snapshot
```

Comment:
Can we just have the inner struct as this now has only one field?

Reply:
I would leave it as is for now. Maybe we will add another field in the future, then we would need to change it back.